### PR TITLE
[spark-md5] Add types

### PR DESCRIPTION
**Pull request by bastienmoulia**

### CODE REVIEW

File: types/spark-md5/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// Type definitions for spark-md5 3.0
+// Project: https://github.com/satazor/js-spark-md5#readme
+// Definitions by: My Self <https://github.com/bastienmoulia>
```
Comment (RyanCavanaugh) [inline types/spark-md5/index.d.ts]: Please fill in your actual name (you may use a psuedonym, just not "My Self")

File: types/spark-md5/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		getState(): State;
+		setState(state: State): State;
+		destroy(): void;
+		static hash(arr: ArrayBufferCopy, raw?: boolean): string;
+	}
+}
+
+export = SparkMD5;
+
+/*declare module "spark-md5" {
```
Comment (RyanCavanaugh) [inline types/spark-md5/index.d.ts]: Remove

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]
Just need two small edits here

**Comment by bastienmoulia** [comment]
@RyanCavanaugh Done!

Do you think the tslint should warn me about the definition name and url when it's not change after the generation with `dts-gen` ? It could have prevented me from making this mistake.

**Comment by RyanCavanaugh** [comment]
Would be nice, though it seems to be a fairly rare error based on experience.

**Pull request merged**